standardtracer/bench_test.go

```diff
@@ -0,0 +1,65 @@
+package standardtracer
+
+import (
+	"fmt"
+	"sync/atomic"
+	"testing"
+)
+
+var tags []string
+
+func init() {
+	tags = make([]string, 1000)
+	for j := 0; j < len(tags); j++ {
+		tags[j] = fmt.Sprintf("%d", randomID())
+	}
+}
+
+type countingRecorder int32
+
+func (c *countingRecorder) RecordSpan(r RawSpan) {
+	atomic.AddInt32((*int32)(c), 1)
+}
+
+func benchmarkWithOps(b *testing.B, numEvent, numTag, numAttr int) {
```

Comment:
I realize we're not quite there yet, but hopefully close to the final version. Benchmarking can expose flaws in the API as well, and I figured you'd be busy enough with all the fallout from stabilization, so I went to give Go a head start.